pandas/io/tests/json/test_pandas.py

```diff
@@ -948,6 +948,58 @@ def test_tz_range_is_utc(self):
         df = DataFrame({'DT': dti})
         self.assertEqual(dfexp, pd.json.dumps(df, iso_dates=True))
 
+    def test_read_jsonl(self):
```

Comment:
can you add some tests that assert ValueError if invalid combination of lines=True and orient?